### DISCUSSION TITLE
Data connections

### DISCUSSION THREAD

**Discussion by oaslsjcnndq** [Budibase/budibase#2932] [Help]
Is it possible to import data that requires connections.

**Comment by mjashanks** [comment]
what do you mean by
> data that requires connections. 

Do you have an example?

**Comment by oaslsjcnndq** [comment]
Example: I already have tables in csv format. I would like to import my already existing tables so that it fits with my new tables.

**Reply by oaslsjcnndq** [reply]
Like I don't want to start from scratch

**Reply by mjashanks** [reply]
Sure, you can import CSVs into a new Budibase table - see the upload button on the create table page: 

![image](https://user-images.githubusercontent.com/3524181/136577543-445666c4-00e6-47a5-96a0-e58cc9c9b3ec.png)

**Comment by oaslsjcnndq** [comment]
Yes but I cannot import with relationship ?

**Reply by mjashanks** [reply]
No, unfortunately not.

If you can get your relationships into a MySql or Postgres DB the Budibase works really well with those.

**Reply by oaslsjcnndq** [reply]
Thanks !
